src/app/effects/user-auth.effect.ts

```diff
@@ -57,7 +57,7 @@ export class UserAuthEffects {
 
     @Effect() storeUserSuccess$ = this.actions$
       .ofType(ActionTypes.STORE_USER_SUCCESS)
-      .map(() => go(['/dashboard']));
+      .map(() => go(['/dashboard/suggestions']));
```

Comment:
@kartikjagdale its routing right ? or is it something else which I am not aware of ?